test_runner/src/main/kotlin/ftl/http/HttpTimeoutIncrease.kt

```diff
@@ -9,6 +9,7 @@ class HttpTimeoutIncrease(credentials: Credentials) : HttpCredentialsAdapter(cre
     override fun initialize(request: HttpRequest?) {
         super.initialize(request)
 
+        request?.headers?.put("X-Server-Timeout", 60)
```

Comment:
Will this apply to all network calls?

Reply:
Should we be worried about it, if it does?

Reply:
> Timeout (in seconds, float value) for the server to finish processing the request. This system param only applies to REST APIs for which client-side timeout is not applicable.

60 second timeout seems incredibly short? Google cloud is quite slow depending on what endpoint we're using.

Reply:
@bootstraponline What value do you think, would be reasonable?